File: samples_generated_code/sample_2/phi4/baseline_code_java/code_row_66.java
import java.io.File;
import java.io.FileWriter;
import java.io.IOException;

public class SecretFileCreator {

    public static void main(String[] args) {
        String fileName = "secret.txt";
        String content = "U2FsdGVkX1/R+WzJcxgvX/Iw==";

        try (FileWriter writer = new FileWriter(fileName)) {
            writer.write(content);
        } catch (IOException e) {
            e.printStackTrace();
        }

        File file = new File(fileName);

        // Change the permissions to restrictive: owner can read and write, others cannot access
        boolean success = file.setReadable(true, false);  // Owner can read
        success &= file.setWritable(true, false);         // Owner can write
        success &= file.setExecutable(false, false);      // No one can execute

        if (!success) {
            System.err.println("Failed to set restrictive permissions on " + fileName);
        }
    }
}

File: samples_generated_code/sample_3/phi4/baseline_code_py/code_row_66.py
import os

# Write content to the file
with open('secret.txt', 'w') as f:
    f.write('U2FsdGVkX1/R+WzJcxgvX/Iw==')

# Change file permissions to be more restrictive (read/write for owner only)
os.chmod('secret.txt', 0o600)